woocommerce/woocommerce-gateway-stripe
# Remove unused WC_Stripe_Old_Settings_UPE_Toggle_Controller

## Changes proposed in this Pull Request:

- Revert the changes introduced by https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1913

These changes were implemented for the old user interface which is no longer in use.

## Testing instructions

Ensure there are no regressions when toggling the Legacy experience:

1. Go to the Stripe extension settings page > Settings tab, at `siteurl/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=settings`
2. Scroll down to "Advanced settings"
3. Toggle the checkbox labeled "Enable the legacy checkout experience" 
4. Save the settings
5. Confirm that:
   - The Legacy experience setting value persists after reloading the page
   - There are no errors in the browser console or in the PHP logs

---


**Post merge**
